### ISSUE TITLE
False positive with non-default imports

### ISSUE THREAD

**Issue by bartenra** [BerniWittmann/vue-unused-components-checker#16]
We sometimes import non-default imports along default import (the Vue component).
 
```ts
import ContentItemSlideOver, { ContentInput } from '@/views/contentpool/slideOver/ContentItemSlideOver.vue';
```

In this case, `ContentItemSlideOver.vue` will be reported as unused.

**Comment by crobison** [comment]
Along the same line, this is also a false positive:
`component: () =>
      import(
        /* webpackChunkName: "event_create" */
        "../views/EventCreate.vue"
      )
  },`
  
  Great tool. It's been very helpful!

**Comment by BerniWittmann** [comment]
Thanks for the findings, will try to fix those issues as soon as i get around to it :)
